App auth method policy GA docs (#20127)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/appmanagementpolicy-post-appliesto.md

---
title: "Assign appliesTo"

> description: "Assign a policy to application or service principal object."

Reviewer: ```suggestion
description: "Assign an appManagementPolicy to an application or service principal object."
```
